components/dear/sources/available-stock-level-change/available-stock-level-change.mjs

```diff
@@ -0,0 +1,32 @@
+import constants from "../../common/constants.mjs";
+import base from "../common/webhooks.mjs";
+
+export default {
+  ...base,
+  name: "Available Stock Level Change",
+  key: "dear-available-stock-level-change",
+  type: "source",
+  description: "Emit new event when available stock level is changed.",
```

Comment:
```suggestion
  description: "Emit a new event when the available stock level changes.",
```

Reply:
Changed "Emit new event" to "Emit a new event" to follow Google's style guide on articles.